server/test/officeTest.js

```diff
@@ -41,4 +41,19 @@ describe('/POST office', () => {
         done();
       });
   });
+});
+
+
+describe('/GET offices', () => {
+  it('it should be able GET all offices', (done) => {
+    chai.request(server)
+      .get('/api/v1/offices')
+      .end((request, response) => {
+        response.should.have.status(200);
+        response.body.should.have.property('statusMessage');
+        response.body.statusMessage.should.equal('Success');
+        response.body.should.be.a('object');
```

Comment:
can you test for the size of the number of offices returned? For example, it's length should be greater than a certain number or equal zero

Reply:
I just did that and I learnt a new skill. thank you very much for pushing me to do more. am greatful